### PR TITLE
Adding thermal conductivities to NEMO output

### PR DESCRIPTION
**Pull request by WallyMaier**
## Proposed Changes
This PR adds the non-constant thermal conductivities to the NEMO output.

## Related Work
Discussion #1888 

## PR Checklist
*Put an X by all that apply. You can fill this out after submitting the PR. If you have any questions, don't hesitate to ask! We want to help. These are a guide for you to know what the reviewers will be looking for in your contribution.*

- [X] I am submitting my contribution to the develop branch.
- [X] My contribution generates no new compiler warnings (try with --warnlevel=3 when using meson).
- [X] My contribution is commented and consistent with SU2 style (https://su2code.github.io/docs_v7/Style-Guide/).
- [ ] I have added a test case that demonstrates my contribution, if necessary.
- [ ] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp), if necessary.

### CODE REVIEW

File: SU2_CFD/src/output/CNEMOCompOutput.cpp [hunk 1]
```diff
@@ -254,6 +254,8 @@ void CNEMOCompOutput::SetVolumeOutputFields(CConfig *config){
     if (nDim == 3)
      AddVolumeOutput("SKIN_FRICTION-Z", "Skin_Friction_Coefficient_z", "PRIMITIVE", "z-component of the skin friction vector");
 
+    AddVolumeOutput("THERMAL_COND_TR",       "Thermal_Cond_tr",       "PRIMITIVE", "thermal tr");
+    AddVolumeOutput("THERMAL__COND_VE",       "Thermal_Cond_ve",       "PRIMITIVE", "thermal vib");
```
Comment (WallyMaier) [inline SU2_CFD/src/output/CNEMOCompOutput.cpp:258]:
```suggestion
    AddVolumeOutput("THERMAL_COND_TR", "Thermal_Cond_tr", "PRIMITIVE", "thermal tr");
    AddVolumeOutput("THERMAL__COND_VE", "Thermal_Cond_ve", "PRIMITIVE", "thermal vib");
```

File: SU2_CFD/src/output/CNEMOCompOutput.cpp [hunk 2]
```diff
@@ -254,6 +254,8 @@ void CNEMOCompOutput::SetVolumeOutputFields(CConfig *config){
     if (nDim == 3)
      AddVolumeOutput("SKIN_FRICTION-Z", "Skin_Friction_Coefficient_z", "PRIMITIVE", "z-component of the skin friction vector");
 
+    AddVolumeOutput("THERMAL_COND_TR", "Thermal_Cond_tr", "PRIMITIVE", "thermal tr");
+    AddVolumeOutput("THERMAL__COND_VE", "Thermal_Cond_ve", "PRIMITIVE", "thermal vib");
```
Comment (pcarruscag) [inline SU2_CFD/src/output/CNEMOCompOutput.cpp:258]: Use more descriptive descriptions please
```suggestion
    AddVolumeOutput("THERMAL_COND_VE", "Thermal_Cond_ve", "PRIMITIVE", "thermal vib");
```

File: SU2_CFD/src/output/CNEMOCompOutput.cpp [hunk 3]
```diff
@@ -332,6 +334,8 @@ void CNEMOCompOutput::LoadVolumeData(CConfig *config, CGeometry *geometry, CSolv
 
   if (config->GetViscous()){
     SetVolumeOutputValue("LAMINAR_VISCOSITY", iPoint, Node_Flow->GetLaminarViscosity(iPoint));
+    SetVolumeOutputValue("THERMAL_COND_TR", iPoint, Node_Flow->GetThermalConductivity(iPoint));
+    SetVolumeOutputValue("THERMAL_COND_VE", iPoint, Node_Flow->GetThermalConductivity_ve(iPoint));
```
Comment (WallyMaier) [inline SU2_CFD/src/output/CNEMOCompOutput.cpp:338]:
```suggestion
    SetVolumeOutputValue("THERMAL_CONDUCTIVITY_TR", iPoint, Node_Flow->GetThermalConductivity(iPoint));
    SetVolumeOutputValue("THERMAL_CONDUCTIVITY_VE", iPoint, Node_Flow->GetThermalConductivity_ve(iPoint));
```

File: SU2_CFD/src/output/CNEMOCompOutput.cpp (hunk 2 above)
Comment (WallyMaier) [inline SU2_CFD/src/output/CNEMOCompOutput.cpp:258]:
```suggestion
    AddVolumeOutput("THERMAL_CONDUCTIVITY_TR", "Thermal_Conductivity_tr", "PRIMITIVE", "thermal tr");
    AddVolumeOutput("THERMAL_CONDUCTIVITY_VE", "Thermal_Conductivity_ve", "PRIMITIVE", "thermal vib");
```

**Review by jtneedels — APPROVED** [review]
This looks good to me. I guess I'm curious if this is something we want to push into the base code though. To address LDK29, they would want this for the surface output.

File: SU2_CFD/src/output/CNEMOCompOutput.cpp (hunk 2 above)
Comment (pcarruscag) [reply SU2_CFD/src/output/CNEMOCompOutput.cpp]: This was not resolved, the descriptions don't say anything meaningful

**Comment by WallyMaier** [comment]
So this will output the surface as well.

Im thinking as more surface effects are looked at, it will become important. Especially when comparing different transport models.  I know I used it extensively for in the RANS models.

File: SU2_CFD/src/output/CNEMOCompOutput.cpp [hunk 4]
```diff
@@ -254,6 +254,8 @@ void CNEMOCompOutput::SetVolumeOutputFields(CConfig *config){
     if (nDim == 3)
      AddVolumeOutput("SKIN_FRICTION-Z", "Skin_Friction_Coefficient_z", "PRIMITIVE", "z-component of the skin friction vector");
 
+    AddVolumeOutput("THERMAL_CONDUCTIVITY_TR", "Thermal_Conductivity_tr", "PRIMITIVE", "thermal tr");
+    AddVolumeOutput("THERMAL_CONDUCTIVITY_VE", "Thermal_Conductivity_ve", "PRIMITIVE", "thermal vib");
```
Comment (WallyMaier) [inline SU2_CFD/src/output/CNEMOCompOutput.cpp:258]:
```suggestion
    AddVolumeOutput("THERMAL_CONDUCTIVITY_TR", "Thermal_Conductivity_tr", "PRIMITIVE", "Translational-rotational thermal conductivity");
    AddVolumeOutput("THERMAL_CONDUCTIVITY_VE", "Thermal_Conductivity_ve", "PRIMITIVE", "Vibrational-electronic thermal conductivity");
```

File: SU2_CFD/src/output/CNEMOCompOutput.cpp (hunk 2 above)
Comment (WallyMaier) [reply SU2_CFD/src/output/CNEMOCompOutput.cpp]: Ahhh my bad.  My pre-coffee brain was not working.  It should be updated now.

**Comment by jtneedels** [comment]
> So this will output the surface as well.
> 
> Im thinking as more surface effects are looked at, it will become important. Especially when comparing different transport models. I know I used it extensively for in the RANS models.

Good point, I was mis-remembering there to be separate AddSurfaceOutput....

Ok sounds good to me then!

**Review by pcarruscag — APPROVED** [review]

**Pull request merged**